modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -138,49 +130,115 @@ The ownCloud windows network drive listener utilizes the SMB notification featur
 +
-.WND server, credentials, and additional mount options
-image::enterprise/external_storage/windows_network_drive/wnd-3.png[WND server, credentials, and additional mount options, width=450]
+image::enterprise/external_storage/windows_network_drive/wnd-gear-icon.png[WND Gear Icon,width=300]
+
+. Your changes are saved automatically.
+
+. Finally, the mount point is created and will look like this example if a user has set it up:
++
+image::enterprise/external_storage/windows_network_drive/wnd-username-pwd.png[WND mount point created,width=500]
+
+NOTE: When you create a new mountpoint using login credentials (session-based), you must log out of ownCloud and then log back in so you can access the share. You only have to do this the first time.
+--
+
+=== Special Settings when using 'Credentials hardcoded in config file'
 
-Your changes are saved automatically.
+The `Config key` to be entered as shown in the screen below, must be taken as described in  xref:enterprise/authentication/enterprise_only_auth.adoc#value-to-be-entered-in-the-mount-point-config-key-field[Value to be entered in the mount point config key field].
 
-NOTE: When you create a new mountpoint using login credentials (session based), you must log out of ownCloud and then log back in so you can access the share. You only have to do this the first time.
+image::enterprise/external_storage/windows_network_drive/wnd-config-key.png[WND mountpoint and hardcoded credentials, width=500]
+
+Note that this authentication method can only be used by ordinary users if the admin hands over the `Config key` created. This is secure, as no password is exposed.
+
+=== Special Settings when Adding a Kerberos Mountpoint
+
+When the xref:enterprise/authentication/kerberos.adoc[Kerberos Authentication] has been set up, a necessary config key xref:configuration/server/config_apps_sample_php_parameters.adoc#a-map-of-servers-with-the-required-kerberos-data[wnd.kerberos.servers] needs to be provided upfront in `config.php`.
+
+* `Kerberos Server ID` +
+This ID can be chosen freely and defines required parameters for the WND Kerberos setup. Multiple ID's with different setups can be created.
+* `ockeytab` +
+This is the keytab file that has been described in xref:enterprise/authentication/kerberos.adoc#keytab-files[Keytab Files].
+* `ocservice` +
+This is the service user that has been described in xref:enterprise/authentication/kerberos.adoc#service-principal-name-spn[Service Principal Name (SPN)].
+
+Enter the `Kerberos Server ID` to define the server and Kerberos credentials as shown in the image below.
+
+image::enterprise/external_storage/windows_network_drive/wnd-kerberos-1.png[WND mountpoint and Kerberos credentials, width=500]
+
+Note that this authentication method can only be used by ordinary users if the admin hands over the `Kerberos Server ID` created. Consider this as very sensitive information like a password.
```

Comment:
I would consider the `Keberos Server ID` as sensitive information. The keytab itself is, but the server id wouldn't help an attacker. The permission are still managed on the server and according the delegation rules.

Reply:
There are two sensitive infos:
* The keytab file itself (different story and covered in the Kerberos docs)
* The server ID not because of an attacker for hacking but to avoid giving access to shares